src/meta/src/rpc/ddl_controller.rs

```diff
@@ -1643,6 +1645,7 @@ impl DdlController {
             table_parallelism,
             max_parallelism.get(),
         );
+        let internal_tables = table_fragments.internal_tables();
```

Comment:
The `internal_tables` field in `CreateStreamingJobContext` will now be complete.